daemon/src/db.rs

```diff
@@ -408,88 +440,119 @@ pub async fn load_cfds_by_oracle_event_id(
     oracle_event_id: BitMexPriceEventId,
     conn: &mut PoolConnection<Sqlite>,
 ) -> anyhow::Result<Vec<Cfd>> {
-    let oracle_event_id = oracle_event_id.to_string();
-
+    let event_id = oracle_event_id.to_string();
```

Comment:
I extended the tests for `load_all_cfds` - but given that these other statements duplicate the complex join SQL code we might also want to add more tests for loading by `oracle_event_id` and `order_id` to make sure we cover someone only changing one of these statements. (unless we can achieve re-using the SQL statement code).

Reply:
Agreed!

Reply:
Added to `test_multiple_cfd_updates_per_cfd()`